src/masternodes/mn_rpc.cpp

```diff
@@ -3086,39 +3089,326 @@ UniValue isappliedcustomtx(const JSONRPCRequest& request) {
+                        p1.second.GetAllTokensAmount() < p2.second.GetAllTokensAmount();
+        });
+    }
+
+    // selecting accounts balances
+    for (auto accIt = foundAccountsBalances.begin(); accIt != foundAccountsBalances.end(); accIt++) {
+#ifdef DEBUG
+        LogPrintf("residualBalances: %s\n", residualBalances.ToString());
+        LogPrintf("candidate: %s\n", accIt->second.ToString());
+#endif
+        // Substract residualBalances and selectedBalances with remainder.
+        // Substraction with remainder will remove tokenAmount from balances if remainder
+        // of token's amount is not zero (we got negative result of substraction)
+        CBalances remainder = residualBalances.SubBalancesWithRemainder(accIt->second.balances);
+#ifdef DEBUG
+        LogPrintf("remainder: %s\n", remainder.ToString());
+#endif
+        // calculate final balances by substraction account balances with remainder
+        // it is necessary to get rid of excess
+        CBalances finalBalances(accIt->second);
+        finalBalances.SubBalances(remainder.balances);
+#ifdef DEBUG
+        LogPrintf("selected balance: %s\n", finalBalances.ToString());
+#endif
+        if (!finalBalances.balances.empty())
+            selectedAccountsBalances.emplace(accIt->first, finalBalances);
+        // if residual balances is empty we found all neccessary token amounts and can stop selecting
+        if (residualBalances.balances.empty())
+            break;
+    }
+
+    const auto selectedBalancesSum = SumAllTransfers(selectedAccountsBalances);
+#ifdef DEBUG
+    LogPrintf("selectedBalancesSum: %s\n", selectedBalancesSum.ToString());
+    LogPrintf("targetBalances: %s\n", targetBalances.ToString());
+#endif
+    if (selectedBalancesSum != targetBalances) {
+        // we have not enough tokens balance to transfer
+        return {};
+    }
+#ifdef DEBUG
+    LogPrintf("selectedAccountsBalances:\n");
+    for (auto &acc : selectedAccountsBalances) {
+        LogPrintf("%s: %s\n", acc.first.GetHex(), acc.second.ToString());
+    }
+#endif
+    return selectedAccountsBalances;
+}
+
+UniValue sendtokenstoaddress(const JSONRPCRequest& request) {
+    CWallet* const pwallet = GetWallet(request);
+
+    RPCHelpMan{"sendtokenstoaddress",
+               "\nCreates (and submits to local node and network) a transfer transaction from your accounts balances (may be picked manualy or autoselected) to the specfied accounts.\n" +
+               HelpRequiringPassphrase(pwallet) + "\n",
+               {
+                    {"from", RPCArg::Type::OBJ, RPCArg::Optional::NO, "",
+                        {
+                            {"address", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "The source defi address is the key, the value is amount in amount@token format. "
+                                                                                     "If obj is empty (no address keys exists) - trying to autoselecting accounts from wallet "
```

Comment:
Should be "then will try to auto-select accounts from wallet"

Reply:
Done